DVTShowcase/AboutView.swift

```diff
@@ -10,12 +10,41 @@ import UIKit
 
 class AboutView: UIView {
 
+    var view:UIView!
```

Comment:
What does this view do, variables should never be named after their type but rather after their function.